packages/sqlserver/sodasql/dialects/sqlserver_dialect.py

```diff
@@ -86,13 +86,13 @@ def sql_connection_test(self):
     def create_connection(self):
         try:
             conn = pyodbc.connect(
-                ('Trusted_Connection=YES;' if self.trusted_connection else '') +
-                'DRIVER={' + self.driver +
-                '};SERVER=' + self.host +
-                ';PORT=' + self.port +
-                ';DATABASE=' + self.database +
-                ';UID=' + self.username +
-                ';PWD=' + self.password)
+                ('Trusted_Connection=NO;' if True else '') +
```

Comment:
Please revert these .. trusted connection is required to make sure that AD SSO works.

Reply:
Yes I will be reversing these, hence the WIP